packages/experimental/AppearanceAdditions/ios/FRNAppearanceAdditions.m

```diff
@@ -105,23 +107,20 @@ - (dispatch_queue_t)methodQueue
 - (void)startObserving {
     _hasListeners = YES;
     
-    // Note that [UITraitCollection currentTraitCollection] always returns the same default trait collection,
-    // presumably because FRNAppearanceAdditions isn't a view, so it never gets updated with the right traitCollection
-    // (which happens when a view gets added to the view hierachy). In order to get the right trait collection,
-    // we need to access a view that's been added to the view hierarchy
-    UIViewController *viewControllerWithInitialTraitCollection = RCTPresentedViewController();
-    UITraitCollection *initialTraitCollection;
-    
-    if (viewControllerWithInitialTraitCollection != nil) {
-        initialTraitCollection = [viewControllerWithInitialTraitCollection traitCollection];
-    } else {
-        initialTraitCollection = [UITraitCollection currentTraitCollection];
+    for (UIWindow *window in RCTSharedApplication().windows) {
+        RCTRootView *rootView = (RCTRootView *)[[window rootViewController] view];
+        
+        if ([rootView isKindOfClass:[RCTRootView class]]) {
```

Comment:
Feels kinda redundant? Maybe line 111 should just be `UIView`?

Reply:
I'm cool with either pattern - if you declare `rootView` as `UIView` you'll just have to cast later anyway.

Reply:
Resolving since I ended up removing this code